spec/change_sets/hyrax/generic_work_change_set_spec.rb

```diff
@@ -362,7 +362,7 @@
 
     # rubocop:disable RSpec/ExampleLength
     it do
-      is_expected.to eq ["created_at", "updated_at", "depositor", "title",
+      is_expected.to include("created_at", "updated_at", "depositor", "title",
```

Comment:
Can you explain the rationale for this change?

Reply:
Per Justin's point, just for fun I switched `'include` to `contain_exactly` and get an error that `import_url` is returned, but not expected, so does this reveal a bug?

```
1) GenericWorkChangeSet#fields should contain exactly "created_at", "updated_at", "depositor", "title", "date_uploaded", "date_modified", "admin_set_id", "state", "proxy_depositor", "on_behalf_of", "arkivo_checksum", "member_of_collection_ids", "member_ids", "thumbnail_id", "representative_id", "label", "relative_path", "resource_type", "creator", "contributor", "description", "keyword", "license", "rights_statement", "publisher", "date_created", "subject", "language", "identifier", "related_url", "source", and "based_near"
     Failure/Error:
       is_expected.to contain_exactly("created_at", "updated_at", "depositor", "title",
                              "date_uploaded", "date_modified", "admin_set_id",
                              "state", "proxy_depositor", "on_behalf_of",
                              "arkivo_checksum", "member_of_collection_ids",
                              "member_ids", "thumbnail_id", "representative_id",
                              "label", "relative_path", "resource_type", "creator",
                              "contributor", "description", "keyword", "license",
                              "rights_statement",
                              "publisher", "date_created", "subject", "language",
                              "identifier", "related_url", "source", "based_near")
     
       expected collection contained:  ["admin_set_id", "arkivo_checksum", "based_near", "contributor", "created_at", "creator", "date_creat...urce_type", "rights_statement", "source", "state", "subject", "thumbnail_id", "title", "updated_at"]
       actual collection contained:    ["admin_set_id", "arkivo_checksum", "based_near", "contributor", "created_at", "creator", "date_creat...urce_type", "rights_statement", "source", "state", "subject", "thumbnail_id", "title", "updated_at"]
       the extra elements were:        ["import_url"]
```

Reply:
test fails, so rather than re-order it seemed better to not require order (since order doesn't matter) ... just testing with contains_exactly

Reply:
@carrickr @jcoyne ... ah, I added 'import_url' in, thinking it was just an omission in the test ... is that a bug? or should it be there?